lib/rfc3164.js

```diff
@@ -3,6 +3,7 @@
 const moment = require('moment-timezone')
 const stringify = require('fast-safe-stringify')
 const through2 = require('through2')
+const EOL = require('os').EOL
```

Comment:
Then let's just have `{ newline: false }` and `{ newline: true }`. When `true`, just append a '\n'. I'm not sure what would happen if a carriage return or carriage return + line feed were added, as using the OS EOL would do in certain circumstances.